docs/en/ingest-management/commands.asciidoc

```diff
@@ -64,6 +121,50 @@ verification.
 `--certificate-authorities <string>`::
 Comma-separated list of root certificates used for server verification.
 
+`--enrollment-token <string>`::
+Enrollment token to use to enroll {agent} into {fleet}. You can use
+the same enrollment token for multiple agents.
+
+`--fleet-server-cert <string>`::
+Certificate to use for exposed {fleet-server} HTTPS endpoint.
+
+`--fleet-server-cert-key <string>`::
+Private key to use for exposed {fleet-server} HTTPS endpoint.
+
+`--fleet-server-es <string>`::
+Start a {fleet-server} process when {agent} is started, and connect to the
+specified {es} URL. You can optionally pass a username and password in the
```

Comment:
The fleet-server-es parameter is required, as noted in prior comment - it is passed along to any Agents that enroll with the given Fleet Server and it is the part of their policy setup which allows them connectivity to send data to ES.

Also, I would avoid citing anything about the prior ability to pass the es-username and es-password as this is being removed from the code for 7.14 if it isn't already.